events/models.py

```diff
@@ -50,7 +50,15 @@ class EventPromotionStatus(Enum):
     in_progress = (20, 'In Progress')
     sent = (30, 'Sent')
     skipped = (40, 'Skipped')
+    event_not_approved = (50, 'Event Not Approved')
+    expired = (60, 'Expired')
 
+"""Statuses that require clearing of the contact list"""
+event_promotion_statuses_for_list_clear = [
```

Comment:
I wonder if this logic makes more sense in organizing hub app rather than here. Kind of related to specific event promo implementation and signal/task logic. Not necessarily universal to all event promo models 🤔